### ISSUE TITLE
Downlink Scheduling

### ISSUE THREAD

**Issue by beneuto** [TheThingsNetwork/nodered-app-node#34]
Proposed enhancement.   Add support for downlink scheduling as per the example below lifted from the TTN API documentation.   Configure as msg inputs so that confirmed and schedule values can be set on a message by message basis.

```
{
  "port": 1,
  "confirmed": false,
  // payload_raw or payload_fields
  "schedule": "replace", // allowed values: "replace" (default), "first", "last"
}
```

**Comment by romeovs** [comment]
I like the idea!

This will take some time however since this library depends on an old version of the `ttn` package, which does not support scheduling to be set. Rewriting it to use the newer version of `ttn` will take a while.

**Comment by johanstokking** [comment]
This should be resolved in `v2.0.2`
